dependencies.gradle

```diff
@@ -26,43 +26,38 @@ ext {
     ]
```

Comment:
Since it has changes related to updated version dependencies and WM usage, please update also the `app version code` and `app version name`:

https://github.com/microsoft/surface-duo-sdk-samples-kotlin/blob/de4442a80ca9805bc0d7b6fd88a7479d0a8e171f/dependencies.gradle#L14